src/components/newsletter/newsletter-sign-up.tsx

```diff
@@ -255,27 +257,9 @@ const NewsletterSignUp = (props: { color: string }) => {
                   grid-column: 2 / span 1;
                 `}
               />
-              <input type="submit" value="Submit" id="submit" />
```

Comment:
can we just hide this behind a flag as well rather than removing it? Will make putting it back in easier e.g. ``` { submitEnabled && <input ... }```

Reply:
not important enough to block this though